frontend/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -255,14 +252,14 @@ const ProgressBox = styled.div`
   display: flex;
   flex-direction: column;
   align-items: center;
-  padding: 30px 30px 30px 30px;
+  padding: 2vh 2vw;
 
   flex: 1 0 50%;
   order: 0;
   flex-grow: 1;
 
   max-width: calc(50% - 16px);
-  margin-bottom: 32px;
+  margin-bottom: 2vh;
```

Comment:
Hmmm this could definitely be saved for a future design bug ticket, but I think it would look cleaner if these progress boxes used margin-top, so that we can align the whole ProgressBoxContainer with the bottom of the screen, instead of a small gap. It looks inconsistent with the bottom of the SessionCards